README.md

```diff
@@ -142,11 +144,11 @@ base layer, and the gorgeous perceptually uniform [cmocean balance](https://matp
 colormap.
 
 <details>
-<summary>🗒 </summary>
+<summary>🗒 click for code</summary>
 
 ```python
 import geovista as gv
-from geovista.pantry import ww3_global_tri
+from geovista.pantry.data import ww3_global_tri
```

Comment:
See https://github.com/bjlittle/geovista/pull/784

Reply:
Oh darn I had forgotten all about that PR. Thanks for spotting!